### PR TITLE
Add UI to Endpoint-Registry

### PR DESCRIPTION
**Pull request by NicoStrazzeri**

### CODE REVIEW

File: src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/Dropdowns/IdTypes.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[{
```
Comment (akiskips) [inline src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/Dropdowns/IdTypes.json:1]: The types are described in AAS-Objects lib. Maybe the menus should reference this lib

File: src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/SemanticProtocols/SemanticProtocols.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+[{
```
Comment (akiskips) [inline src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/SemanticProtocols/SemanticProtocols.json:1]: Depending on functionality consider moving to a test folder

**Review by akiskips — COMMENTED** [review]
added some comments regarding mock data and dropdown menus. 
Main consideration would be if the aas-objects lib can be used for default menu values

File: src/ts/cmd/endpoint-registry/src/services/registry/routes.ts [hunk 3]
```diff
@@ -323,5 +325,22 @@ export default [
       }
 
     }
-    }
+  },
+
+  // {
```
[the remark this answers is not in the document]
Comment (NicoStrazzeri) [reply src/ts/cmd/endpoint-registry/src/services/registry/routes.ts]: [Removed commented out code to improve readability](https://github.com/SAP/i40-aas/pull/182/commits/8477f381a1df3cdbc5b361653ad33601eb2dbf70)

File: src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/SemanticProtocols/SemanticProtocols.json (hunk 2 above)
Comment (NicoStrazzeri) [reply src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/SemanticProtocols/SemanticProtocols.json]: [Unused Json deleted](https://github.com/SAP/i40-aas/pull/182/commits/654733785652a1ba7a5aeeedc48c4925385f8ec3)

**Review by heldic — APPROVED** [review]

**Review by etoster — APPROVED** [review]

**Review by akiskips — APPROVED** [review]

**Pull request merged**